Unix/cli/cli_c.c

```diff
@@ -2491,31 +2491,31 @@ const MI_Char USAGE[] = MI_T(
 "    id\n"
 "        Send identify request.\n"
 "    gi NAMESPACE INSTANCENAME\n"
-"        Peform a CIM [g]et [i]nstance operation.\n"
+"        Perform a CIM [g]et [i]nstance operation.\n"
```

Comment:
ditto. Note that I've told the spell-checker to ignore `[g]et` and friends, so if there's a typo there, I won't notice.